admin/client.go

```diff
@@ -201,7 +201,7 @@ func (c *Client) ListChangeSummaries(
 		return nil, err
 	}
 
-	seq := changes[0].ServerSeq()
+	seq := changes[0].ServerSeq() - 1
```

Comment:
> Is there any problem with changing this code? CC) @chacha912



> To explain why I've implemented like this, when there's a `--backend-snapshot-with-purging-changes` flag, the changes of the front are deleted, so I want to create a history based on the changes that the ListChanges succeeded in loading from db . I thought there would be no problem because the change is imported from the db based on GetChangesRange in ListChanges.
> 
> I'd appreciate it if you point out something that I missed.

I updated it little bit, check here please!
CC) @chacha912

Reply:
I like it. 
It didn't occur to me that I can get start-sequence by `changes[0].ServerSeq()`.  Thanks :)